README.md

```diff
@@ -138,6 +138,14 @@ alias ...='cd ../../../'
 alias ....='cd ../../../../'
 alias .....='cd ../../../../'
 ```
+<a id="youtube-mp3"></a>
+#### Download music from youtube video
+You will need mp3-lame library and youtube-dl utility.
+```sh
+alias youtube-mp3="youtube-dl --extract-audio --audio-format mp3"
```

Comment:
What if we shrink it to `yt-mp3`

Reply:
I do prefer the longer alternative youtube-mp3. It makes more clear what this tool is for - while yt-mp3 dosen't give many informations about it. But if you prefer of course I can set it to `yt-mp3`